gsms.py

```diff
@@ -205,5 +287,42 @@ def main() -> None:
                                 'custom location.')
 
 
+def get_win_path(folder_id) -> str:
+    """
+    Function to resolve Windows UUID folders into their absolute path
+
+    Parameters
+    ----------
+    folder_id: str
```

Comment:
These should be the name and type, followed by a description. Please see the documentation (https://numpydoc.readthedocs.io/en/latest/format.html#parameters).

```rst
folder_id : str
    The Windows UUID folder to find the path for.
```

Note: the description should also be a proper sentence, i.e. end with a `.`
Note: the space between `folder_id` and `:`

Reply:
Forgot that description ya. The space between colon I copied the style from the `stopwatch`